### ISSUE TITLE
Remove subjectHeadings from contributors

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/lobid-resources#519]
There 76 contributions by subjectHeadings: http://lobid.org/resources/search?q=contribution.agent.type:SubjectHeading. A few glances on resources only resulted in persons which are also typed as subjectHeadings.

**Comment by acka47** [comment]
This doesn't look right. There are several cases of an author not having a GND ID and (probably) the same person being subject with GND ID, see e.g. http://lobid.org/resources/BT000081623 & http://lobid.org/resources/BT000109314. Althiugh, the original data is already problematic the transformation to RDF could also be improved, I think.

**Comment by ChristophEwertowski** [comment]
I think that my original thought was that it doesn't make much sense that a `SubjectHeading` can create some work or help with publishing. [Definition](http://id.loc.gov/ontologies/bibframe/contribution.rdf) of `contribution`: 
>  \<skos:definition>Agent and its role in relation to the resource.</skos:definition> 

And `agent` is [defined](http://id.loc.gov/ontologies/bibframe/agent.rdf) as the following:
> Entity associated with a resource or element of description, such as the name of the entity responsible for the content or of the publication, printing, distribution, issue, release or production of a resource.

Also I concur with acka47.

**Comment by dr0i** [comment]
Only 3 left.

**Comment by acka47** [comment]
I will notify verbund.

**Comment by dr0i** [comment]
Wait - I don't think it's a catalogue error. They resue the `902` field, one time _with_ an gnd-ID, sometimes _not_. I think this allowed. It's sometimes hard to transform. I think it must be done correcting the morph. Just a gut feeling, not anlyzed in depth yet.

**Comment by acka47** [comment]
Yes, looking at this a bit more I had similar thoughts and did not send the email. Also, I think  we would have to adjust some more things in the morph because even the good-looking lobid entries are missing information from the catalogue. However, as this issue only affects three entries and the other things havenÄt been reported by anybody else, we should move it to backlog.

**Comment by TobiasNx** [comment]
Could be fixed with ALMA-Fix.

**Comment by TobiasNx** [comment]
@dr0i can you check if the alma-index is build

**Comment by TobiasNx** [comment]
Seems to be no problem for Fix anymore: https://test.lobid.org/resources/search?q=contribution.agent.type:SubjectHeading
